src/providers/log.provider.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-next-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import { Provider } from '@loopback/context';
+import { ParsedRequest } from '@loopback/core';
```

Comment:
coding style - no spaces inside/around curly braces

It would be best if you could run prettier for all newly added files to keep consistent formatting.